### PR TITLE
Retrieve and Store Job Data from DCI and Quay in SQL Database

### PR DESCRIPTION
**Pull request by shimritproj**
[](https://issues.redhat.com/browse/CNFCERT-1059 )
This PR implements functionality to retrieve job and component data from DCI and Quay and store it in an SQL database. 
The data from Quay includes:

- aggregated.Datetime
- aggregated.Count
- aggregated.Kind

The data from DCI includes:

- job.ID
- commit
- job.CreatedAt
- totalSuccess
- totalFailures
- totalErrors
- totalSkips

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module CERTSUITE-OVERVIEW
```
Comment (sebrandon1) [inline go.mod:1]: I think this should be `module github.com/shimritproj/certsuite-overview`

File: quay.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	"log"
+	"time"
+
+	_ "github.com/go-sql-driver/mysql"
+
+	quay "github.com/sebrandon1/go-quay/lib"
+)
+
+const (
+	bearerToken = "wdSLOoT8K3pa4BQhsTnWxLSobTWMNF4X67p4VmU1"
```
Comment (sebrandon1) [inline quay.go:13]: This value can't be checked/hardcoded in, needs to be passed in via config or something.

Comment (shimritproj) [reply quay.go]: thanks! I agree!!! I created config folder and there I put bearerToken, clientID, namespace and repository.

**Review by sebrandon1 — APPROVED** [review]

**Comment by sebrandon1** [comment]
Looks good.  Let's merge this in and we can do some next steps.

**Pull request merged**